packages/core/src/crypto/jose/jwk/P_256Jwk.ts

```diff
@@ -0,0 +1,102 @@
+import type { JwkJson } from './Jwk'
+
+import { TypedArrayEncoder, Buffer } from '../../../utils'
+import { KeyType } from '../../KeyType'
+import { JwaCurve, JwaKeyType } from '../jwa'
+import { JwaSignatureAlgorithm } from '../jwa/alg'
+
+import { Jwk } from './Jwk'
+import { compress, expand } from './ecCompression'
+import { hasKty, hasCrv, hasX, hasY, hasValidUse } from './validate'
+
+export class P_256Jwk extends Jwk {
+  public readonly x: string
+  public readonly y: string
+
+  public constructor({ x, y }: { x: string; y: string }) {
+    super()
+
+    this.x = x
+    this.y = y
+  }
+
+  public get kty() {
+    return JwaKeyType.EC as const
+  }
+
+  public get crv() {
+    return JwaCurve.P_256 as const
+  }
+
+  public get keyType() {
+    return KeyType.P256
+  }
+
+  // NOTE: this is the compressed variant. We should add support for the uncompressed variant.
+  public get publicKey() {
+    const publicKeyBuffer = Buffer.concat([TypedArrayEncoder.fromBase64(this.x), TypedArrayEncoder.fromBase64(this.y)])
```

Comment:
Shouldn't this be base64url?

Reply:
the fromBase64 method supports both... A bit confusing I know.